src/Data/Models/MRIUploadDTO.php

```diff
@@ -0,0 +1,194 @@
+<?php declare(strict_types=1);
+
+/**
+ * PHP Version 7
+ *
+ * @category Imaging
+ * @package  Main
+ * @author   Loris Team <loris-dev@bic.mni.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\Data\Models;
+
+/**
+ * This class defines an MRIUploadDTO which is an immutable representation of
+ * an MRIUpload object.
+ *
+ * @category Imaging
+ * @package  Main
+ * @author   Loris Team <loris-dev@bic.mni.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class MRIUploadDTO implements \LORIS\Data\DataInstance
+{
+    private $mri_upload_id;
```

Comment:
Not necessary, but these could just use constructor property promotion.

Reply:
Is this ok ?
https://github.com/aces/Loris/pull/9154/commits/4d6eeb1e2c2d4d99a8590538de4c88521783695b